src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,35 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Is responsible for handling commands
+ */
+public class Command {
+    /**
+     * Created when the command does not belong to any other types of command.
+     */
+    public Command() {}
+
+    /**
+     * Returns whether this command terminates the program.
+     * @return An indicator that helps the program know when to stop (default is keep running)
+     */
+    public boolean isExit() {
+        return false;
+    }
+
+    /**
+     * Executes the command.
```

Comment:
Maybe it would be better to give a short explanation on how the command is executed?